collections/0xethereals/meta.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "0xEthereals",
+  "inscription_icon": "0xaa41dc8c1e8b8cbba347924646a2a9189c32fb2203cfceac703c0bee86ad380a",
```

Comment:
This is fine.  just don't forget that you are free to set a custom logo by adding a 'logo.png' file in this dir if you want.

Reply:
Sorry it took a long time. We decided to leave icon like this for consistency.